mcs/build/library.make

```diff
@@ -289,15 +289,9 @@ endif
 # TODO: depend on all *.sources (except tests) for now and figure out how to list only needed files later
 PROFILE_sources = $(filter-out %test.dll.exclude.sources %test.dll.sources, $(wildcard *.sources))
 
-ifneq "x" "x$(PROFILE_RUNTIME)"
-GENSOURCES_RUNTIME=$(PROFILE_RUNTIME)
-else
-GENSOURCES_RUNTIME=MONO_PATH="$(GENSOURCES_LIBDIR)$(PLATFORM_PATH_SEPARATOR)$$MONO_PATH" $(RUNTIME)
-endif
-
 sourcefile = $(depsdir)/$(PROFILE_PLATFORM)_$(PROFILE)_$(LIBRARY_SUBDIR)_$(LIBRARY).sources
-$(sourcefile): $(PROFILE_sources) $(PROFILE_excludes) $(depsdir)/.stamp $(GENSOURCES_CS)
```

Comment:
We don't track dependencies out of mcs/class and the tool is build before we enter that folder anyway. It could be improved but not sure it's worth it